features/step_definitions/folder_steps.rb

```diff
@@ -0,0 +1,37 @@
+Given(/^I already have the Git autocompletion symlink$/) do
+  FileUtils.rm File.expand_path('~/.config/fish/completions/git.fish')
+  FileUtils.symlink 'foo',
+                    File.expand_path('~/.config/fish/completions/git.fish')
+end
+
+
+Given(/^I have a different Git autocompletion file$/) do
+  FileUtils.rm File.expand_path('~/.config/fish/completions/git.fish')
+  IO.write File.expand_path('~/.config/fish/completions/git.fish'),
+           'existing Git autocompletion data'
+end
+
+
+Given(/^I have no fish autocompletion folder$/) do
+  FileUtils.rm_rf File.expand_path('~/.config/fish/completions')
+end
+
+
+Given(/^I have an empty fish autocompletion folder$/) do
+  suppress do
```

Comment:
This is part of Rails' [ActiveSupport library](http://guides.rubyonrails.org/active_support_core_extensions.html), which extends Ruby with even more awesomeness.

Here is the [documentation for suppress](http://www.rubydoc.info/gems/activesupport/4.2.0/Kernel:suppress)

Here is where we load it: https://github.com/Originate/git-town/blob/master/features/support/env.rb#L1